pages/common/hardhat.md

```diff
@@ -0,0 +1,36 @@
+# hardhat
+
+> A development environment for Ethereum software.
+> More information: <https://hardhat.org>.
+
+- List available commands (or create a new project if no configuration exists):
+
+`hardhat`
+
+- Compile the project and build all artifacts:
+
+`hardhat compile`
+
+- Run a user-defined script after compiling the project:
+
+`hardhat run {{path/to/script.js}}`
+
+- Run Mocha tests:
+
+`hardhat test`
+
+- Run one or more specific space-separated test files:
+
+`hardhat test {{path/to/file1.js}} {{path/to/file2.js}}`
+
+- Start a local Ethereum JSON-RPC for development:
```

Comment:
```suggestion
- Start a local Ethereum `JSON-RPC` for development:
```

Reply:
I don't think this needs to be in backticks as it's not a command, unless something has changed since I last contributed. 🤔

Reply:
The contributing guides say we should use it for "technical" terms. I think it makes the sentence a bit more readable by ''isolating" words which interrupt the reading flow.

Reply:
Personally I'm not sure JSON-RPC would count for this, as [according to the style guide](https://github.com/tldr-pages/tldr/blob/ec4a9a4ee60700decb2d78c7361a43c238bb22f6/contributing-guides/style-guide.md#special-cases) backticks should be used for paths, extensions, or commands. But happy to see what other people think. 👍🏻

Reply:
Yeah, JSON-RPC is an acronym, so it doesn't make sense to enclose it in backticks.

But this doesn't make sense anyway, because JSON-RPC is likely to be a standard - so how can you start an "Etherium JSON-RPC"?

Reply:
It's an Ethereum JSON-RPC node, I guess I should include the node bit.